lib/tasks/data.rake

```diff
@@ -75,4 +76,137 @@ namespace :aleftng do
       end
     end
   end
+
+  def find_and_replace_in_text(find, replace, text)
```

Comment:
Tato metoda robi iba to, ze prehadze svoje argumenty a miesto ```gsub``` sa to vola ```find_and_replace_in_text```. Nahradit zavolanim ```gsub``` v mieste pouzitia.

(Ak je cielom, sledovanie vykonavania, ako naznacuje zakomentovany vypis, na to predsa sluzi debugger.)